tempesta_fw/http.h

```diff
@@ -290,6 +290,7 @@ typedef struct {
 /* URI has form http://authority/path, not just /path */
 #define TFW_HTTP_URI_FULL		0x000400
 #define TFW_HTTP_NON_IDEMP		0x000800
+#define TFW_HTTP_SUSPECTED		0x001000
```

Comment:
Like for `Conn_Suspected` (please see the comment below), I believe the name is misleading as the decision has been made already. Perhaps, simply `TFW_HTTP_MSG_BAD`? This name may mean either of a message being filtered out, or being malicious, or being erroneous.

Reply:
I don't agree: `TFW_HTTP_MSG_BAD` doesn't say what `bad` is? Is it corrupted or malicious message? I'm OK with `suspected`.

Reply:
What I was trying to say was that in my opinion that there's nothing that is `suspected` at this point. The decision had been made, the message *IS* now either filtered out, malicious, or erroneous. Yet, all of these messages are now marked as suspected. Due to this decision the client connection will be closed.

So I didn't understand what it's suspected of?

Suspicion is when we don't know what it is yet, it's either this or that, and there are further actions to come up with more detailed knowledge.  But it doesn't work that way in the code. Again, the decision had been made at the time this flag is set, with the HTTP error response code and the error message. There's nothing that is suspected. Just the opposite, everything is pretty clear and definite.

Just a thought. :-)